src/main/java/pw/chew/chewbotcca/commands/DiffCommand.java

```diff
@@ -62,44 +66,68 @@ protected void execute(CommandEvent event) {
             String[] args = event.getArgs().split(" ");
 
             if (args.length != 2) {
-                event.reply("Invalid number of arguments provided. Please provide valid role IDs for this server.");
+                event.reply("Invalid number of arguments provided. Please provide valid role names.");
                 return;
             }
 
-            Role base = event.getGuild().getRoleById(args[0]);
+            String baseString = args[0].toLowerCase();
```

Comment:
Thats something new to me. I didn't know about spaces between names and roles. I am thinking of using a seperator like "/ " instead of space between names or roles. If roles have the same name , this wouldn't work ... But how many discord users have developer mode enabled or even know about this, so that they can copy id of a role? Thats a problem.